augur/subsample.py

```diff
@@ -240,6 +241,17 @@ def _get_search_paths(config_file: str, from_cli: List[str]):
         Path.cwd(),
     ]
 
+    if from_env := os.environ.get('AUGUR_SEARCH_PATHS'):
+        if from_cli:
+            print_err(dedent(f"""\
+                WARNING: Both the environment variable AUGUR_SEARCH_PATHS
+                and the command line argument --search-paths are set.
+                Only the environment variable will be used."""))
```

Comment:
Ah, I think it's more conventional (though not universal) for command line options to take precedence over environment variables, i.e. swap the order here.

Conventional precedence from least to most goes: config file → env var → command line option.

Reply:
Thanks, I'll swap the order.